teamcity/changelog.d/18122.fixed

```diff
@@ -0,0 +1 @@
+[AGENT-11701] Fix handling of deleted build config
```

Comment:
```suggestion
Fix handling of deleted build config
```
I don't think we typically post the Jira issue in the changelog, especially since it's a public repo and we're the only ones with access to this board 👍